pkgs/development/embedded/openocd-rp2040/default.nix

```
@@ -0,0 +1,68 @@
+{ stdenv
+, fetchgit
+, lib
+, which
+, git
+, autoconf
+, automake
+, texinfo
+, libtool
+, pkg-config
+, hidapi
+, libftdi1
+, libusb1
+, libgpiod
+}:
```

```suggestion
{ lib
, stdenv
, fetchgit
, pkg-config
, hidapi
, libftdi1
, libusb1
, which
, libtool
, autoconf
, automake
, texinfo
, git
, libgpiod
}:
```
> Arguments should be listed in the order they are used, with the exception of lib, which always goes first.

See [nixpkgs-manual](https://nixos.org/manual/nixpkgs/stable/#sec-syntax)

---

thanks